activerecord/lib/active_record/connection_adapters/abstract_adapter.rb

```diff
@@ -161,6 +161,14 @@ def schema_creation
         SchemaCreation.new self
       end
 
+      # Returns an array of +Column+ objects for the table specified by +table_name+.
+      def columns(table_name) # :nodoc:
+        table_name = table_name.to_s
+        column_definitions(table_name).map do |field|
```

Comment:
In the squlite3 adapter this was called `table_strucutures` this is breaking the tests right now.